contracts/keep3r-liquidity-manager/Keep3rLiquidityManagerUserLiquidityHandler.sol

```diff
@@ -9,16 +9,32 @@ import '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
 import './Keep3rLiquidityManagerParameters.sol';
 
 interface IKeep3rLiquidityManagerUserLiquidityHandler {
-  event DepositedLiquidity(address indexed _depositor, address _recipient, address _lp, uint256 _amount);
+  event LiquidityFee(uint256 _liquidityFee);
+
+  event LiquidityMin(address _liquidity, uint256 _liquidityFee);
+
+  event DepositedLiquidity(address indexed _depositor, address _recipient, address _lp, uint256 _amount, uint256 _fee);
 
   event WithdrewLiquidity(address indexed _withdrawer, address _recipient, address _lp, uint256 _amount);
 
+  function liquidityFee() external view returns (uint256 _liquidityFee);
+
+  function feeReceiver() external view returns (address _feeReceiver);
+
+  function liquidityMinAmount(address _liquidity) external view returns (uint256 _minAmount);
+
   function liquidityTotalAmount(address _liquidity) external view returns (uint256 _amount);
 
   function userLiquidityTotalAmount(address _user, address _lp) external view returns (uint256 _amount);
 
   function userLiquidityIdleAmount(address _user, address _lp) external view returns (uint256 _amount);
 
+  function setLiquidityFee(uint256 _liquidityFee) external;
+
+  function setFeeReceiver(address _feeReceiver) external;
+
+  function setMinAmount(address _liquidity, uint256 _minAmount) external;
+
```

Comment:
shouldnt this be in `parameters` maybe ? idk really, just checking if it makes sense.

Reply:
I wanted to assign variables where they are used. min has been moved to userJobLiq